docker/boot_notebook.bash

```diff
@@ -54,6 +54,18 @@ cat > .jupyter_config.json <<EOF
 }
 EOF
 
+# cookie security options for cross domain
+cat > "$HOME/.jupyter/jupyter_notebook_config.py" <<EOF
+c.JupyterHub.tornado_settings = {
+    'cookie_options': {'SameSite': 'Strict', 'Secure': True}
+}
+
+c.NotebookApp.tornado_settings = {
+    'cookie_options': {'SameSite': 'Strict', 'Secure': True}
+}
+c.NotebookApp.disable_check_xsrf = True
```

Comment:
Can you add a link to the doc of these options? This way it is simple to understand their meaning and allows easy access to doc to tune them in the future

Reply:
done